net-libs/tox/files/tox-bootstrapd.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
```

Comment:
Why this? Unless I'm missing something, you don't use anything but `ExecStart=` anymore.